tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -614,6 +614,7 @@ public void RunWithInterpreterEnabled ([Values (false, true)] bool isRelease)
 
 			proj = new XamarinAndroidApplicationProject () {
 				IsRelease = isRelease,
+				AotAssemblies = false, // Release defaults to Profiled AOT for .NET 6
```

Comment:
Any of the tests with this comment, they relied on it being a non-AOT build to pass.